task/seed.js

```diff
@@ -14,10 +14,10 @@ Promise.all([
     database.sequelize.model('trial').bulkCreate(seedData.trial),
     database.sequelize.model('question_option').bulkCreate(seedData.questionOption),
     database.sequelize.model('question_template').bulkCreate(seedData.questionTemplate),
-    database.sequelize.model('join_questions_and_options').bulkCreate(seedData.joinQuestionsAndOptions),
     database.sequelize.model('survey_template').bulkCreate(seedData.surveyTemplate),
-    database.sequelize.model('join_trials_and_surveys').bulkCreate(seedData.joinTrialsAndSurveys),
-    database.sequelize.model('join_surveys_and_questions').bulkCreate(seedData.joinSurveysAndQuestions)
+    // database.sequelize.model('join_trials_and_surveys').bulkCreate(seedData.joinTrialsAndSurveys),
```

Comment:
yes I do know which of these stages relate to which trials because in stage table we have a trial id associated to each stage.
I will fill out the details for this table in seed-data file